projects/microphysics/argo/end_to_end.py

```diff
@@ -121,25 +121,21 @@ def _label_args(labels):
 
 
 def set_prognostic_emulation_model(
```

Comment:
Over the past couple weeks, I've noticed that this function adds a layer of indirection that made it harder for me to understand the configuration. It appears that you needed to make some changes for this PR, suggesting the abstraction is not very robust.

What do you think about delete this and move the appropriate settings into a yaml file? I think copy/pasting yamls will be clearer in this case.